### PR TITLE
ENH: plot prediction curve over scatter in GLMGamResults.plot_partial

### PR DESCRIPTION
**Pull request by aglebov**
Currently, when called with `cpr=True`, `GLMGamResults.plot_partial` will plot the scatter of residuals over the fitted curve:
image

The change plots the curve over the scatter and reduces the marker size:
image

### CODE REVIEW

File: statsmodels/gam/generalized_additive_model.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def plot_partial(self, smooth_index, plot_se=True, cpr=False,
             If plot_se is true, then the confidence interval for the linear
             prediction will be added to the plot.
         cpr : bool
-            If cpr (component plus residual) is true, the a scatter plot of
+            If cpr (component plus residual) is true, them a scatter plot of
```
Comment (josef-pkt) [inline statsmodels/gam/generalized_additive_model.py:322]: typo `then`

**Comment by josef-pkt** [comment]
Thanks for the PR, 
LGTM except replacing one typo with another typo

amend and force push would get the cleanest commit

**Comment by aglebov** [comment]
> Thanks for the PR, LGTM except replacing one typo with another typo

I've fixed the typo. Do you want to merge this change?

**Comment by josef-pkt** [comment]
Thanks for the pull request.
merging

**Pull request merged**